src/app/builder/builder.component.html

```diff
@@ -122,32 +132,78 @@
       <h4>Copy and Use Your Embed Code</h4>
 
       <p>Copy the code for one of the following players, and place it somewhere on your webpage.</p>
+      <section class="builder-step" id="step-4a" *ngIf="props">
```

Comment:
Only reason was the other "step-X" ids in the html, wanted to keep it consistent. Happy to change it to a class attribute though, esp if we are moving it up to the top outside of the builderForm.